src/chainparams.cpp

```diff
@@ -146,6 +147,7 @@ class CTestNetParams : public CChainParams {
         consensus.BlockV9TallyHeight = 399120;
         consensus.BlockV10Height = 629409;
         consensus.BlockV11Height = 1301500;
+        consensus.powLimit = uint256S("0000ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
```

Comment:
You should retain the commenting on why these values are what they are (and what they are in compact notation...)